### DISCUSSION TITLE
Unable to get the xdrip watchface from the nightly releases at all

### DISCUSSION THREAD

**Discussion by Npavkov** [NightscoutFoundation/xDrip#3416] [Bug reports]
Since last year I have only been able to install the wear os component of xdrip from the RAW directory in the APK. When I install this version the watch CONSTANTLY shows that xdrip is updating....ALL DAY LONG. I wanted to try to get the current version of wear os component from the recent nightly builds but I have not been successful. Does ANYONE know how to get the current wear component to install on watch from the new nightly builds?? OR How to stop xdrip from constantly updating the wear os component on the watch?

**Comment by Navid200** [comment]
https://github.com/NightscoutFoundation/xDrip/discussions/3261#discussioncomment-7983726

**Reply by Npavkov** [reply]
how do I extract it using apktool???
